app/src/main/java/com/imaginaryrhombus/proctimer/ui/timer/TimerFragment.kt

```diff
@@ -109,5 +111,13 @@ class TimerFragment : Fragment() {
             val pickerFragment = TimerPickerFragment()
             pickerFragment.show(fragmentManager, "PickerDialog")
         }
+
+        privacy.setOnClickListener {
+            // TODO : Remote Config に移動する.
+            // TODO : ナビゲーションドロワーを使用して配置したい.
+            val intent = Intent(Intent.ACTION_VIEW, Uri.parse("https://pagehosting-d362c.firebaseapp.com/proctimer/privacy-poricy.html"))
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/fizxive/ProcTimer/blob/ce13a8f47cb152ecfa360a6edb2d384d211799f1/app/src/main/java/com/imaginaryrhombus/proctimer/ui/timer/TimerFragment.kt#L118"></span>Exceeded max line length (100) (cannot be auto-corrected)</td>
    </tr>
  </tbody>
</table>